app/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "peruse",
-  "productName": "Peruse",
+  "name": "safe-browser",
```

Comment:
@bochaco @joshuef Should this be `safeBrowser` instead?
Otherwise we'd need to use `window["safe-browser"]`.

Reply:
I would agree, or would 'safe_browser' work and be preferable?